packages/sanity/src/structure/panes/document/documentPanel/header/perspective/DocumentPerspectiveMenu.tsx

```diff
@@ -26,7 +26,7 @@ export function DocumentPerspectiveMenu(props: {documentId: string}): JSX.Elemen
   const {currentGlobalBundle} = usePerspective()
   const bundles = useMemo(() => data ?? [], [data])
 
-  const existsInBundle = bundles.some((bundle) => bundle.slug === getBundleSlug(documentId))
+  const existsInBundle = getBundleSlug(documentId) === currentGlobalBundle?.slug
```

Comment:
Was the misaligned logic here surfaced by the test by any chance? 👀 
Least if it was you can say that all the toil was worth it...

Reply:
I have to admit I don't recall 🥲